src/packages/user/current-user/theme/current-user-theme-user-profile-app.element.ts

```diff
@@ -50,15 +50,28 @@ export class UmbCurrentUserThemeUserProfileAppElement extends UmbLitElement {
 		return html`
 			<uui-box headline="Theme">
 				<uui-tag slot="headline" look="placeholder">Experimental</uui-tag>
-				<uui-select label="Select theme" .options=${this._themes} @change=${this.#onThemeChange}></uui-select>
```

Comment:
Can we make the change in the UI Library instead? Then will the select be styled correctly across the entire backoffice?